datanode/sqlstore/testdata/marketdata.csv

```diff
@@ -1,551 +1,551 @@
-53035a92e297d29b6729bcd5cf032e675efb0e3df8ff4a0f0e876edde7faf3eb,2022-02-11 10:02:36.086299 +00:00,2022-02-11 10:02:37.000000 +00:00,1000034006,1000029421,1,1000034006,3,1000029421,1,1000034006,3,1000031713,1000031713,13,0,0,0,0,TRADING_MODE_CONTINUOUS,AUCTION_TRIGGER_UNSPECIFIED,AUCTION_TRIGGER_UNSPECIFIED,32501105195,50000000000,null,227441856154953.0000000000000000,"[{""party"": ""af2bb48edd738353fcd7a2b6cea4821dd2382ec95497954535278dfbfff7b5b5"", ""equity_like_share"": ""1"", ""average_entry_valuation"": ""50000000000"", ""average_score"": ""123""}]",STATE_ACTIVE
-53035a92e297d29b6729bcd5cf032e675efb0e3df8ff4a0f0e876edde7faf3eb,2022-02-11 10:02:37.395648 +00:00,2022-02-11 10:02:38.000000 +00:00,1000034006,1000029421,1,1000032398,2,1000029421,1,1000032398,2,1000030909,1000030909,13,0,0,0,0,TRADING_MODE_CONTINUOUS,AUCTION_TRIGGER_UNSPECIFIED,AUCTION_TRIGGER_UNSPECIFIED,32501105195,50000000000,null,182509877807322.0000000000000000,"[{""party"": ""af2bb48edd738353fcd7a2b6cea4821dd2382ec95497954535278dfbfff7b5b5"", ""equity_like_share"": ""1"", ""average_entry_valuation"": ""50000000000"", ""average_score"": ""123""}]",STATE_ACTIVE
-53035a92e297d29b6729bcd5cf032e675efb0e3df8ff4a0f0e876edde7faf3eb,2022-02-11 10:02:38.716713 +00:00,2022-02-11 10:02:39.000000 +00:00,1000034006,1000029421,1,1000034006,2,1000029421,1,1000034006,2,1000031713,1000031713,11,0,0,0,0,TRADING_MODE_CONTINUOUS,AUCTION_TRIGGER_UNSPECIFIED,AUCTION_TRIGGER_UNSPECIFIED,32501105195,50000000000,null,152177640218531.0000000000000000,"[{""party"": ""af2bb48edd738353fcd7a2b6cea4821dd2382ec95497954535278dfbfff7b5b5"", ""equity_like_share"": ""1"", ""average_entry_valuation"": ""50000000000"", ""average_score"": ""123""}]",STATE_ACTIVE
-53035a92e297d29b6729bcd5cf032e675efb0e3df8ff4a0f0e876edde7faf3eb,2022-02-11 10:02:39.997006 +00:00,2022-02-11 10:02:41.000000 +00:00,1000034006,1000029421,1,1000034006,2,1000029421,1,1000034006,2,1000031713,1000031713,11,0,0,0,0,TRADING_MODE_CONTINUOUS,AUCTION_TRIGGER_UNSPECIFIED,AUCTION_TRIGGER_UNSPECIFIED,32501105195,50000000000,null,327667631512506.800000000000000,"[{""party"": ""af2bb48edd738353fcd7a2b6cea4821dd2382ec95497954535278dfbfff7b5b5"", ""equity_like_share"": ""1"", ""average_entry_valuation"": ""50000000000"", ""average_score"": ""123""}]",STATE_ACTIVE
-53035a92e297d29b6729bcd5cf032e675efb0e3df8ff4a0f0e876edde7faf3eb,2022-02-11 10:02:41.315951 +00:00,2022-02-11 10:02:42.000000 +00:00,1000034006,1000029421,1,1000034006,2,1000029421,1,1000034006,2,1000031713,1000031713,11,0,0,0,0,TRADING_MODE_CONTINUOUS,AUCTION_TRIGGER_UNSPECIFIED,AUC
```

Comment:
Setting them all to zero isn't sufficient. If the `entities.MarketData.MarketGrowth` is _**NOT**_ being saved into the database, when we populate a `entities.MarketData` struct when reading from the database the `MarketGrowth` field would default to `0` and the test would still pass.

Reply:
#NotAllZeroes There's a couple of them (literally 2 I think :P) that have non-zero values